examples/advanced/job_api/sklearn/kmeans_script_executor_higgs.py

```diff
@@ -117,24 +118,29 @@ def split_higgs(input_data_path, input_header_path, output_dir, site_num, sample
     # ScatterAndGather also expects an "aggregator" which we define here.
     # The actual aggregation function is defined by an "assembler" to specify how to handle the collected updates.
     # We use KMeansAssembler which is the assembler designed for k-Means algorithm.
-    aggregator = CollectAndAssembleAggregator(assembler_id=job.as_id(KMeansAssembler()))
+    assembler_id = job.to_server(KMeansAssembler())
```

Comment:
Should specify the "id" param so the generated id is more descriptive:
assembler_id = job.to_server(KMeansAssembler(), id="assembler")